rules/falco_rules.yaml

```diff
@@ -2365,7 +2369,12 @@
      sysdig/sysdig, falcosecurity/falco,
      fluent/fluentd-kubernetes-daemonset, prom/prometheus,
      ibm_cloud_containers,
-     public.ecr.aws/falcosecurity/falco)
+     public.ecr.aws/falcosecurity/falco, velero/velero,
+     quay.io/jetstack/cert-manager-cainjector, weaveworks/kured,
+     ingress-controller/controller, quay.io/prometheus-operator/prometheus-operator,
```

Comment:
I haven't found `ingress-controller/controller` as a public image. Can you check this or remove it pls.